💄chore: update styling to be consistant

TGTGamer opened the pull request.

TGTGamer committed:
:busts_in_silhouette: Add @gavindsouza as a contributor

TGTGamer committed:
:busts_in_silhouette: Add @maxprilutskiy as a contributor

TGTGamer committed:
:busts_in_silhouette: Add @atlassian-compass[bot] as a contributor

TGTGamer committed:
:busts_in_silhouette: Add @ChillarAnand as a contributor

TGTGamer committed:
:busts_in_silhouette: Add @FHenry as a contributor

TGTGamer committed:
:busts_in_silhouette: Add @nasirrr as a contributor

TGTGamer committed:
:busts_in_silhouette: Add @ankush as a contributor

TGTGamer committed:
:busts_in_silhouette: Add @kennethsequeira as a contributor

TGTGamer committed:
:busts_in_silhouette: Add @restyled-commits as a contributor

TGTGamer committed:
:busts_in_silhouette: Add @sagar-bhogayata as a contributor

TGTGamer committed:
:busts_in_silhouette: Add @zaid2229 as a contributor

TGTGamer committed:
:busts_in_silhouette: Add @mymi14s as a contributor

TGTGamer committed:
:busts_in_silhouette: Add @systonium as a contributor

TGTGamer committed:
:busts_in_silhouette: Add @NagariaHussain as a contributor

TGTGamer committed:
:busts_in_silhouette: Add @HUMENTH as a contributor

TGTGamer committed:
:busts_in_silhouette: Add @robproject as a contributor

TGTGamer committed:
:busts_in_silhouette: Add @roquegv as a contributor

TGTGamer committed:
:busts_in_silhouette: Add @mohsinalimat as a contributor

TGTGamer committed:
:busts_in_silhouette: Add @Alchez as a contributor

TGTGamer committed:
:busts_in_silhouette: Add @yemikudaisi as a contributor

TGTGamer committed:
:busts_in_silhouette: Add @nextchamp-saqib as a contributor

TGTGamer committed:
:busts_in_silhouette: Add @dependabot[bot] as a contributor

TGTGamer committed:
:busts_in_silhouette: Add @mergify[bot] as a contributor

TGTGamer committed:
:busts_in_silhouette: Add @TMateSoft as a contributor

TGTGamer committed:
:arrow_up: chore(contributing.md): update contributors list

Update the "contributing.md" file to include additional contributors.

Signed-off-by: Jonathan Stevens <jonathan.stevens@eventiva.co.uk>

TGTGamer committed:
style: update code styling

Signed-off-by: Jonathan Stevens <jonathan.stevens@eventiva.co.uk>

EventivaLtd reviewed (dismissed):
<!--undefined: Conventions-->




SemanticEmoji Conditions failed - Please title your pull request using a combination of the valid options:
Example: ":bug: fix(context): Fixing a bug in context"

Gitmoji Options: 
⚡️ || :zap: === Improve performance.,
🐛 || :bug: === Fix a bug.,
🚑️ || :ambulance: === Critical hotfix.,
✨ || :sparkles: === Introduce new features.,
💄 || :lipstick: === Add or update the UI and style files.,
🔒️ || :lock: === Fix security issues.,
⬇️ || :arrow_down: === Downgrade dependencies.,
⬆️ || :arrow_up: === Upgrade dependencies.,
📌 || :pushpin: === Pin dependencies to specific versions.,
📈 || :chart_with_upwards_trend: === Add or update analytics or track code.,
➕ || :heavy_plus_sign: === Add a dependency.,
➖ || :heavy_minus_sign: === Remove a dependency.,
🔧 || :wrench: === Add or update configuration files.,
🌐 || :globe_with_meridians: === Internationalization and localization.,
✏️ || :pencil2: === Fix typos.,
⏪️ || :rewind: === Revert changes.,
📦️ || :package: === Add or update compiled files or packages.,
👽️ || :alien: === Update code due to external API changes.,
💥 || :boom: === Introduce breaking changes.,
🍱 || :bento: === Add or update assets.,
♿️ || :wheelchair: === Improve accessibility.,
💬 || :speech_balloon: === Add or update text and literals.,
🗃️ || :card_file_box: === Perform database related changes.,
🚸 || :children_crossing: === Improve user experience / usability.,
📱 || :iphone: === Work on responsive design.,
🥚 || :egg: === Add or update an easter egg.,
⚗️ || :alembic: === Perform experiments.,
🔍️ || :mag: === Improve SEO.,
🏷️ || :label: === Add or update types.,
🚩 || :triangular_flag_on_post: === Add, update, or remove feature flags.,
🥅 || :goal_net: === Catch errors.,
💫 || :dizzy: === Add or update animations and transitions.,
🗑️ || :wastebasket: === Deprecate code that needs to be cleaned up.,
🛂 || :passport_control: === Work on code related to authorization, roles and permissions.,
🩹 || :adhesive_bandage: === Simple fix for a non-critical issue.,
👔 || :necktie: === Add or update business logic.

Semantic Options: 
bug, chore, opt, optimisation, style, maint, maintenance, ref, refactor, revert, dep, deprecated, removal, docs, documentation, feat, enhance, feature, enhancement, fix

For more information on Semantic Commit Messages, please see https://www.conventionalcommits.org/en/v1.0.0/
For more information on gitmoji, please visit https://gitmoji.dev/

----------

This message will be automatically updated when you make this change

TGTGamer committed:
🐛 fix(.github/ISSUE_TEMPLATE): update Code of Conduct link

Updated the link to the project's Code of Conduct in the issue templates to point to the main branch instead of the develop branch. This change ensures that contributors agree to follow the most up-to-date code of conduct.

Signed-off-by: Jonathan Stevens <jonathan.stevens@eventiva.co.uk>
